src/Qir/Runtime/lib/QIR/bridge-rt.ll

```diff
@@ -233,7 +233,6 @@ define dllexport %Array* @__quantum__rt__array_copy(%Array* %.ar, i1 %force) {
 
 ; TODO: This bridge isn't cross-platform!
 ; it works on Windows but on Linux %args ends up not being a valid pointer.
-declare void @DebugLogPtr(i8*)

```

Comment:
nit: now that `DebugLogPtr` has been removed, we can remove the commented out call sites below.

Reply:
I'm not sure I understand.
The `DebugLog[Ptr]()` have relocated to `TestUtils.{h,c}pp`. And I don't see any calls to them (they exist in case anybody needs to use them, but nobody calls/links them at the moment).

Reply:
There are commented out calls later in this file, or at least [one here](https://github.com/microsoft/qsharp-runtime/pull/634/files/ecae5566d3bdecd699e88cbe999a4904fef154a6#diff-c9b697f7183ad4664bf2423e5205f04b07fda1ef2654d756872430292a246118R241) that I can't comment on directly because it is too far from a changed line.

Reply:
I suppose because the function hasn't been fully removed we could leave the  commented out lines behind, but it looks to me like those were debug lines accidentally left in by earlier investigation, and so there is no reason to leave them in.